### PR TITLE
[ISSUE #10583]修复权限

### LINKED ISSUE #10583: 开启鉴权后，/v1/core/cluster/nodes在未登录的情况下依然能访问
版本：2.2.3
开启鉴权后，/v1/core/cluster/nodes在未登录的情况下依然能访问

### PR DESCRIPTION
**Pull request by 985492783**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change
修复某些接口没有经过鉴权
fix #10583 
## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [x] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [x] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigOpsController.java [hunk 1]
```diff
@@ -78,6 +80,7 @@ public String updateLocalCacheFromStore() {
     }
     
     @PutMapping(value = "/log")
+    @Secured(resource = Constants.OPS_CONTROLLER_PATH, action = ActionTypes.WRITE, signType = SignType.CONFIG)
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigOpsController.java:83]: type should be `CONSOLE`

File: config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigOpsController.java [hunk 2]
```diff
@@ -70,6 +71,7 @@ public ConfigOpsController(DumpService dumpService) {
      * Manually trigger dump of a local configuration file.
      */
     @PostMapping(value = "/localCache")
+    @Secured(resource = Constants.OPS_CONTROLLER_PATH, action = ActionTypes.WRITE, signType = SignType.CONFIG)
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigOpsController.java:74]: type should be `CONSOLE`

File: core/src/main/java/com/alibaba/nacos/core/controller/NacosClusterController.java [hunk 3]
```diff
@@ -103,6 +110,7 @@ public RestResult<String> getHealth() {
      */
     @Deprecated
     @PostMapping(value = {"/report"})
+    @Secured(resource = Commons.NACOS_CORE_CONTEXT + "/cluster", action = ActionTypes.WRITE, signType = SignType.CONSOLE)
```
Comment (KomachiSion) [inline core/src/main/java/com/alibaba/nacos/core/controller/NacosClusterController.java:113]: This api is called by servers, you should make sure the api can be called success after you add this.

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: core/src/main/java/com/alibaba/nacos/core/controller/NacosClusterController.java (hunk 3 above)
Comment (985492783) [reply core/src/main/java/com/alibaba/nacos/core/controller/NacosClusterController.java:113]: method reportByHttp will call AuthHeaderUtil.addIdentityToHeader(header);
image

**Pull request merged**